Changelog.md

```diff
@@ -24,8 +24,11 @@ Bugfixes:
  * SMTChecker: SSA control-flow did not take into account state variables that were modified inside inlined functions that were called inside branches.
  * Yul: Properly register functions and disallow shadowing between function variables and variables in the outside scope.
 
+Build System:
+ * soltest.sh: allow environment variable `SOLIDITY_BUILD_DIR` to specify build folder. Add `--help` usage
 
-
+Documentation:
```

Comment:
Please remove. We usually do not mention changes to the documentation.

Reply:
done.